packages/location-field/src/index.tsx

```diff
@@ -566,6 +569,7 @@ const LocationField = ({
         // Create and return the option menu item
         const option = (
           <TransitStopOption
+            id={`${optionIdPrefix}${itemIndex}`}
```

Comment:
Is there any way to extract this string out to prevent it having to be written out multiple times? I'd also prefer to have the `-` in the template than in the prefix

Reply:
I was on the fence with this, but thanks, fixed in https://github.com/opentripplanner/otp-ui/pull/548/commits/863460865b553d65d70bdbd65972d8e6ee95cf21.